app/gui2/src/util/ast/__tests__/abstract.test.ts

```diff
@@ -406,3 +407,89 @@ if (false) {
   })
 }
  */
+
+test('full file IdMap round trip', () => {
+  const content = `from Standard.Base import all
+from Standard.Table import all
+from Standard.Database import all
+from Standard.AWS import all
+import Standard.Visualization
+
+main =
```

Comment:
Please move that out to a separate file (maybe to a `__tests__/fixtures` directory).